openedx/features/partners/urls.py

```diff
@@ -0,0 +1,10 @@
+"""
+The urls for philu features app.
+"""
+from django.conf.urls import url
+
+from .views import dashboard
+
+urlpatterns = [
+    url(r"^partner/(?P<slug>[0-9a-z]+)/$",  dashboard, name="partner_url"),
```

Comment:
Since this is supposed to be generic, are we sure that `0-9a-z` is all we want? Maybe we could reconsider this to include `-` hyphens and `_` underscores?

Reply:
you can also use this regex for slug `(?P<slug>[-\w]+)`